ui/app/components/task-sub-row.hbs

```diff
@@ -0,0 +1,79 @@
+<tr class="task-sub-row"
+	{{keyboard-shortcut
+		enumerated=true
+		action=(action "gotoTask" @allocation this.task)
+	}}
+>
+	<td colspan={{@namespan}}>
+		/
+		<LinkTo @route="allocations.allocation.task" @models={{array @allocation this.task}}>
+			{{this.task.name}}
+		</LinkTo>
+		{{!-- TODO: in-page logs --}}
+		{{!-- <FlightIcon @name="logs" /> --}}
```

Comment:
`nit`: Did you mean to commit this?

Reply:
Yes, commented here: https://github.com/hashicorp/nomad/pull/14592#pullrequestreview-1109508840